Kernel/FileSystem/SysFS/Subsystems/Kernel/Variables/IntegerVariable.cpp

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2023, Mark Douglas <dmarkd@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/FileSystem/SysFS/Subsystems/Kernel/Variables/IntegerVariable.h>
+#include <Kernel/Sections.h>
+#include <Kernel/Tasks/Process.h>
+
+namespace Kernel {
+
+ErrorOr<void> SysFSSystemIntegerVariable::try_generate(KBufferBuilder& builder)
+{
+    return builder.appendff("{}\n", value());
+}
+
+ErrorOr<size_t> SysFSSystemIntegerVariable::write_bytes(off_t, size_t count, UserOrKernelBuffer const& buffer, OpenFileDescription*)
+{
+    MutexLocker locker(m_refresh_lock);
+    // Note: We do all of this code before taking the spinlock because then we disable
```

Comment:
Is this really the case where page fault will not work? I know `SysFSSystemStringVariable` mentioned this, but maybe we could just use `safe_memcpy` here (and in `SysFSSystemStringVariable` too).

Reply:
Also, I think this comment is old and not relevant even for `SysFSSystemStringVariable` anymore too.